thirdparty/download-thirdparty.sh

```diff
@@ -279,6 +279,14 @@ if test "x$PATCH_COMPILER_RT" == "xtrue"; then
     echo "Finished patching $COMPILER_RT_SOURCE"
 fi
 
+cd $TP_SOURCE_DIR/$COMPILER_RT_SOURCE
```

Comment:
Hi, zhengguo, LLVM has been removed from Doris dependency. So I think we can not compile LLVM now.